src/sentry/rules/conditions/event_frequency.py

```diff
@@ -618,21 +626,21 @@ def batch_query_hook(
         self, group_ids: set[int], start: datetime, end: datetime, environment_id: int
     ) -> dict[int, int]:
         batch_percents: dict[int, int] = defaultdict(int)
-        groups = Group.objects.filter(id__in=group_ids)
-        project_id = groups[0].project.id
-        session_count_last_hour = self.get_session_count(project_id, environment_id, start, end)
+        groups = Group.objects.filter(id__in=group_ids).values_list("id", "type", "project")
+        project_ids = Project.objects.filter(id=groups[0][2]).values_list("id", flat=True)
```

Comment:
Is there a way to get the project id in the group query above? I wasn't able to figure it out